go.mod

```
@@ -181,4 +197,6 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 
 replace github.com/lightninglabs/pool/auctioneerrpc => ./auctioneerrpc
 
-go 1.19
+go 1.21.4
+
+toolchain go1.22.2
```

Hmm shouldn't this be:
```suggestion
toolchain go1.22.3
```

Although that change should maybe be in the last commit?

---

Was generated automatically, but yes you're right. Updated.